### ISSUE TITLE
isReadonly Property Not Setting Disabled Attribute

### ISSUE THREAD

**Issue by sky-gregory** [gunjandatta/gd-bs#10]
When setting the isReadonly property on a Components.IFormControlPropsCheckbox control, the html attribute disabled="disabled" isn't being added to the control. Also, for some reason checkboxes don't have the Bootstrap styling. Not sure if that's only my environment though. 

```javascript
{
    size: 2,
    control: {
        label: "Roll Up:",
        name: "Test",
        isReadonly: true,
        type: Components.FormControlTypes.Checkbox,
        onValidate: control => {           
            return {
                isValid: true
            };
        }
    } as Components.IFormControlPropsCheckbox
}
```

**Comment by gunjandatta** [comment]
Fixed in v1.3.8. Let me know if you have any issues w/ it.

**Comment by gunjandatta** [comment]
Please re-open if this is still an issue.
